### ISSUE TITLE
[Accessibility]A11y_VSCode_KeyboardShortcuts_Keyboard: Trim Trailing WhiteSpace(Ctrl+K Ctrl+X) shortcut Command is not working

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52672]
**Environment Details:** 
VSCode Version : 1.24.1

**Additional Details:**
MAS Violated: MAS2.1.1

**Repro Steps:**
1. Launch VS Code.
2. Open Keyboard Shortcuts.

**Actual:**
 Trim Trailing WhiteSpace (Ctrl+K Ctrl+X) shortcut Command is not working

**Expected:**
All the Keyboard command should work.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
User will not be able to use the particular shortcut command.

**MAS Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Attachment for Reference: 
![trimtrailing](https://user-images.githubusercontent.com/39204909/41893468-75444280-7939-11e8-8991-9a4934d89e98.JPG)

[A11y VSCode Keyboard Shortcuts TrimTrailingWhitespace.pptx](https://github.com/Microsoft/vscode/files/2131984/A11y.VSCode.Keyboard.Shortcuts.TrimTrailingWhitespace.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chrmarti** [comment]
Could this be #52681? There is no screencast attached.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Video is already there in "Attachment for Reference" section, also attached the screenshot. Please refer the video for more clarification.

**Comment by chrmarti** [comment]
The recording shows a comma after the whitespace that you expect to be trimmed. "Trim Trailing Whitespace" refers to whitespace after any other characters on the same line with no other characters to the right of the whitespace. The behavior shown in the recording is expected due to that comma to the right.
